tests/steps/account_v2_steps.py

```diff
@@ -18,7 +18,7 @@ def fund_account_address(
     )
     signed_payment = context.wallet.sign_transaction(payment)
     context.app_acl.send_transaction(signed_payment)
-    dev_mode_wait_for_confirmation(context, payment.get_txid(), 10)
+    transaction.wait_for_confirmation(context.app_acl, payment.get_txid(), 10)
```

Comment:
I think waiting one round would be sufficient for dev mode:

```suggestion
    transaction.wait_for_confirmation(context.app_acl, payment.get_txid(), 1)
```

Reply:
@algochoi I agree with the suggestion though I think it ought to be applied across-the-board (in other SDKs too).  https://github.com/algorand/py-algorand-sdk/pull/365/commits/a37f2407999dcc04cf9de83a51c377a9c2e1094c brings in the feedback.

Can you resolve if it looks good?